src/go/k8s/apis/core/v1alpha1/redpandacluster_webhook.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package v1alpha1 represent Custom Resource definition of the vectorized.io core group
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var redpandaclusterlog = logf.Log.WithName("redpandacluster-resource")
```

Comment:
nit: Since is a package level logger, couldn't we just go with `log` ?

Reply:
Agree. Changed to `log`.